h2o-core/src/main/java/hex/grid/GridSearch.java

```diff
@@ -139,6 +138,29 @@ private GridSearch(Key<Grid> gkey, HyperSpaceWalker<MP, ?> hyperSpaceWalker) {
         gridSearch(grid);
         tryComplete();
       }
+
+      /**
+       * If export_checkpoints_dir is defined in parameters, saves the whole Grid into that
+       * folder with checkpoints.
+       *
+       * @param caller the task invoking this method (which may
+       */
+      @Override
+      public void onCompletion(CountedCompleter caller) {
```

Comment:
should we maybe save the grid after every succeful model train? I think that would be best

Reply:
Done. Thanks.

Reply:
where? I cant find it

Reply:
Was not pushed yet. It's pushed now. Sorry.